.github/workflows/cargo-audit.yml

```diff
@@ -40,24 +27,17 @@ jobs:
           - macos-latest
           #- windows-latest # TODO(tarcieri): debug Windows build timeouts
         toolchain:
-          - 1.57.0 # MSRV
+          #- 1.57.0 # MSRV
```

Comment:
For some reason it's not working when an explicit Rust release version is provided.

I opened an upstream issue: https://github.com/Swatinem/rust-cache/issues/57